### ISSUE TITLE
Tracking du modèle texte via MLFlow

### ISSUE THREAD

**Issue by JoffreyLGT** [JoffreyLGT/e-commerce-mlops#115]
Mettre en place Airflow sur le script du modèle texte.

**Comment by JoffreyLGT** [comment]
Je reprend le sujet et vais mettre en place un code semblable à #114.

**Comment by JoffreyLGT** [comment]
Uodate: code is ready. However, I face the same issue I had with the image model when serving the model using mlflow serve. I'm investigating.

**Comment by JoffreyLGT** [comment]
I figured out what the issue is: MLFlow autolog generates requirements files automatically. However, on MacOS, it seems to had only `tensorflow-macos` without the main package : `tensorflow`.

I thought this wouldn't be an issue since we are using the local environment to serve the model. Turns out it is! Which means two things:
- #127 doesn't work as it is. It was working on my machine after so many attempts that I didn't realise it wouldn't work on a fresh environment.
- Even when using local environment and with venv activated, `mlflow serve` might still be pulling packaged or create its own space. I need to figure this out.

As for now, I'm going to use Poetry to generate a requirements.txt file when we are logging the model.
